pkg/apis/componentconfig/types_pluginargs.go

```diff
@@ -78,3 +78,23 @@ type RemoveDuplicatesArgs struct {
 	Namespaces        *api.Namespaces
 	ExcludeOwnerKinds []string
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type LowNodeUtilizationArgs struct {
+	metav1.TypeMeta
+
+	ThresholdPriority                 *int32
+	ThresholdPriorityClassName        string
+	NodeResourceUtilizationThresholds *api.NodeResourceUtilizationThresholds
+}
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type HighNodeUtilizationArgs struct {
+	metav1.TypeMeta
+
+	ThresholdPriority                 *int32
+	ThresholdPriorityClassName        string
```

Comment:
Same as above, not needed any longer.

Reply:
i update at below, may i click resolve now?